### DISCUSSION TITLE
Quantize To semitones on the oscillators

### DISCUSSION THREAD

**Discussion by Radiofloyd2351** [publicsamples/Oi-Grandad#16] [Ideas]
I love this plugin to bits! 
I do hope there can eventually be a quantization that is a little more user friendly though, as it's quite hard to adjust pitch without access to the usual semitones and cents. I think it would be nice if you could give us the option between tempo quantizing and pitch quantizing (the way a quantizer on modular, for example, might behave).

**Comment by RickTaylor79** [comment]
Thanks, I'm thrilled people like it. Never thought it would get this popular.

This is a great idea, and definitely something I'd like to implement. For starters I'll add a second, semitone pitch control. This is simple  and should make basic adjustments easier. Then I'll look into quantising modulation signals, it's probably just a case of adapting the tempo sync to scales (you may have noticed, that when synced it plays kind of "scale" when modulated).

Does this sound good?

**Reply by Radiofloyd235** [reply]
Yeah, that sounds great!

**Reply by RickTaylor79** [reply]
Okay, so the latest version has an additional pitch control locked to semitones. The quantiser thing is certainly doable, but it's gonna take a bit more time. Thanks again for the suggestion, definitely on the list
